src/components/Text.tsx

```diff
@@ -154,92 +122,72 @@ export function BadgeText({ children }: OnlyChildrenProp) {
   return <span className={badgeText}>{children}</span>
 }
 
-interface LinkTextProps {
+interface LinkTextProps extends OnlyChildrenProp {
   url: string
-  children: ReactNode
+  bold?: boolean
+  title?: string
+  color?: TTextColor
   gradientFrom?: TGradientColorStops
```

Comment:
What are the gradient options? Can we limit them by using an enum and a switch inside the classnames function?